examples/complete/main.tf

```diff
@@ -0,0 +1,61 @@
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 5.0.0"
+
+  name                 = local.vpc_name
+  cidr                 = var.vpc_cidr
+  private_subnets      = var.private_subnets
+  azs                  = var.availability_zones
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+
+  tags = {
+    Name        = "example-vpc"
+    Environment = "dev"
+  }
+}
+
```

Comment:
This PR is not complete. As discussed with @bhanu-koya-nttd on slack, you need to first create the ecs_appmesh_platform and ecs_appmesh_ingress module before you could provision the ecs_appmesh_app module